data-prepper-pipeline-parser/src/main/java/org/opensearch/dataprepper/pipeline/parser/rule/RuleEvaluator.java

```diff
@@ -64,7 +65,7 @@ public RuleEvaluatorResult isTransformationNeeded(PipelinesDataFlowModel pipelin
                             .build();
                 }
             } catch (FileNotFoundException e) {
-                LOG.error("Template File Not Found");
+                LOG.error("Template File Not Found for {}", PLUGIN_NAME);
```

Comment:
Actually, you don't even need this catch block. You can remove it. And then go back to using a local `pluginName` variable.